core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/Profiler.scala

```diff
@@ -326,6 +326,8 @@ class Profiler(hadoopConf: Configuration, appArgs: ProfileArgs, enablePB: Boolea
     val stageMetrics = collect.getStageLevelMetrics
     val wholeStage = collect.getWholeStageCodeGenMapping
     val sparkRapidsBuildInfo = collect.getSparkRapidsInfo
+    val sqlToStages = collect.getSQLToStage
```

Comment:
no need to define that variable here. It implies calling this method very early then holding the information until it is passed to the ApplicationSummaryInfo constructor.

Reply:
thanks, reverted this change.